lib/locales/en.yml

```diff
@@ -15,13 +15,12 @@ en:
   chapter_review: Chapter Review
   learning_objectives: Learning Objectives
   stepwise_step_label: Step
-  references: References
+  references: References # is this used somewhere outside eoc?
```

Comment:
thoughts? 😄

Reply:
Yes, in American Government (I guess :thinking:) and in Polish Psychology section References is in EOB. We use there `BakeReferences` module.